index.html

```diff
@@ -685,6 +685,10 @@ <h3>Discover SDKs</h3>
                         <img src="./img/java-icon.svg" alt="go icon" />
                         <a target="_blank" href="https://github.com/KingGorrin/znn_sdk_java/">Java <i class="fas fa-arrow-right"></i></a>
                       </div>
+                      <div class="funding__row"></div>
+                        <img src="./img/php-icon.png" alt="php icon" />
```

Comment:
Looks like this image is missing from the PR

Reply:
Yep that was my fault - added.